lib/daru/core/query.rb

```diff
@@ -39,7 +39,7 @@ def inspect
 
       class << self
         def apply_scalar_operator operator, data, other
-          BoolArray.new data.map { |d| !!d.send(operator, other) }
+          BoolArray.new data.map { |d| !!d.send(operator, other) unless d.nil? }
```

Comment:
Missing data is not just `nil`. What if the `Vector` contains strings combined with a `Float::NaN`? This will fail in that case.